lib/active_fedora/file/fedora_attributes.rb

```diff
@@ -0,0 +1,39 @@
+module ActiveFedora::File::FedoraAttributes
```

Comment:
I like this module. I'm not so keen on the name. I don't think that Fedora has anything do do with whether or not a File has a mime_type, name, digest, etc. Can we just have `AF::File::Attributes`?